charts/trento-server/charts/trento-web/templates/configmap.yaml

```diff
@@ -14,4 +14,5 @@ data:
   ALERT_RECIPIENT: "{{ .Values.alerting.recipient }}"
   PROMETHEUS_URL: "http://{{ .Release.Name }}-{{ .Values.global.prometheus.name }}"
   AMQP_URL: "amqp://trento:trento@{{ .Release.Name }}-{{ .Values.global.rabbitmq.name }}:{{ .Values.global.rabbitmq.servicePort }}"
-  CHARTS_ENABLED: "{{ .Values.chartsEnabled }}"
\ No newline at end of file
+  CHARTS_ENABLED: "{{ .Values.chartsEnabled }}"
+  TRENTO_DOMAIN: "{{ .Values.trentoDomain }}"
```

Comment:
@rtorrero we don't have any `trentoDomain` in `charts/trento-server/charts/trento-web/values.yaml`.

Is that desired or should we have some sort of default value there?

Reply:
It is now a mandatory var, I can't think of a good default because it needs to be user-specific. I'd rather have it fail than put an incorrect value here.

Reply:
sure, makes sense